WoWsShipBuilder.Web/Features/UserSettings/UserSettings.razor

```diff
@@ -92,9 +92,18 @@
             </MudText>
         </MudItem>
         <MudItem xs="12">
-            <MudStack AlignItems="AlignItems.Start">
-                <MudCheckBox @bind-Checked="@storeBuildOnShare" LabelPosition="LabelPosition.Start" Color="Color.Primary" Label="Save build on share"/>
-            </MudStack>
+           <MudCheckBox @bind-Checked="@storeBuildOnShare" LabelPosition="LabelPosition.End" Color="Color.Primary" Label="@Localizer.SimpleAppLocalization(nameof(Translation.SettingsWindow_SaveBuildOnShare))"/>
+        </MudItem>
+        <MudItem xs="12">
+            <MudText Typo="Typo.h6">
+                @Localizer.SimpleAppLocalization(nameof(Translation.SettingsWindow_FittingToolSelector))
+            </MudText>
+            <MudCheckBox @bind-Checked="@AppSettings.FittingToolSelectorClosedByDefault" LabelPosition="LabelPosition.End" Color="Color.Primary" Label="@Localizer.SimpleAppLocalization(nameof(Translation.SettingsWindow_ClosedByDefault))"/>
```

Comment:
do not bind directly on the app settings, use a local variable to allow detection of changes.